.github/workflows/end-to-end-test-data-generator.yml

```diff
@@ -18,6 +18,21 @@ on:
         required: false
         type: string
         description: Comma-separated list of report years. The list of years needs to be the same lenght as the list of dbkeys.
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
```

Comment:
`choice` isn't supported for `workflow_call` for some reason, but someone would have to add this job to the prod deploy workflow for it to run there.